pkgs/servers/x11/quartz-wm/default.nix

```
@@ -1,39 +1,47 @@
 {
   lib,
   stdenv,
-  fetchurl,
-  xorg,
+  fetchgit,
+  autoreconfHook,
   pixman,
   pkg-config,
+  util-macros,
+  libXinerama,
+  libAppleWM,
+  xorgproto,
+  libXrandr,
+  libXext,
 }:
 
-let
-  version = "1.3.1";
-in
-stdenv.mkDerivation {
+stdenv.mkDerivation (finalAttrs: {
   pname = "quartz-wm";
-  inherit version;
-  src = fetchurl {
-    url = "http://xquartz-dl.macosforge.org/src/quartz-wm-${version}.tar.xz";
-    sha256 = "1j8zd3p7rhay1s3sxq6anw78k5s59mx44xpqla2ianl62346a5g9";
+  version = "1.3.2";
+
+  src = fetchgit {
+    url = "https://gitlab.freedesktop.org/xorg/app/quartz-wm.git";
+    tag = "quartz-wm-${finalAttrs.version}";
```

Could make use of fetchFromGitLab

```suggestion
  src = fetchFromGitLab {
    domain = "gitlab.freedesktop.org";
    group = "xorg";
    owner = "app";
    repo = "quartz-wm";
    tag = "quartz-wm-${finalAttrs.version}";
```